recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+{% set pythrift_version = "0.16.*" %}
+{% set thrift_version = "0.16.*" %}
+# heavydb 6.0.0 is not LLVM 12 ready
+{% set llvm_version = "11" %}
+
+package:
+  name: heavydb-ext
+  version: {{ version }}
+
+source:
+  url: https://github.com/heavyai/heavydb/archive/v{{ source_version }}.tar.gz
+  sha256: a36d91227135e9ec8641b5f46e80927aaafce117f3446933dd7f392c9659996e
+  patches:
+    - 0003-Fix-HeavyDBE-build.patch
+
+build:
+  number: {{ number }}
+  skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+
+outputs:
+
+  - name: heavydb-common
+    version: {{ version }}
+    script: build-heavydb.sh  # [linux64]
+    script: build-heavydb-common.bat  # [win]
+    build:
+      string: {{ build_string }}
+      # cpu-only heavydb-common for cuda is required only for
+      # building cuda-enabled heavydb
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0")]
+      skip: true  # [osx]
+      skip: true  # [win and cuda_compiler_version != "None"]
+      run_exports:
+        - {{ pin_subpackage('heavydb-common',  max_pin='x.x.x') }}
+      ignore_run_exports:
+        - arrow-cpp {{ arrow_version }}
+        - blosc
+        - double-conversion
+        - fmt
+        - geos
+        - glog
+        - libarchive
+        - libgdal
+        - libllvm{{ llvm_version }}
+        - libpng
+        - librdkafka
+        - libstdcxx-ng
+        - libthrift
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably...
+        - {{ compiler('c') }}
+        # go required for ThirdParty/generate_cert
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - ninja
+        - maven
+      host:
+        # heavydb-common does not depend on build_ext, arrow-cpp
+        # (and other dependencies) is required just for the presence
+        # so that one could run cmake for generating *.bc, and other
+        # common data files.
+        - arrow-cpp ={{ arrow_version }}=*cpu
+        - bisonpp  # [unix]
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - double-conversion
+        - flex  # [unix]
+        - fmt
+        - geos
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libevent  # [win]
+        - libgdal
+        - libpng
+        - librdkafka
+        - tbb  # [win]
+        - tbb-devel  # [win]
+        - thrift-cpp {{ thrift_version }}
+        - winflexbison  # [win]
+    test:
+      commands:
+        # Test installation
+        # doc
+        - test -f ${PREFIX}/share/doc/heavyai/LICENSE.md  # [unix]
+        - if not exist %PREFIX%\share\doc\heavyai\LICENSE.md exit 1  # [win]
```

Comment:
Unix-style locations on Windows should use `LIBRARY_PREFIX` as the base path. This also applies to all the other instances (test and build scripts) where this might be happening.

Also, how come Unix needs `install_base` but Windows does not? Does that mean that not using `install_base` is supported? Because in that case, we should try that before merging.